src/Reader.tsx

```diff
@@ -31,14 +33,46 @@ export function Reader({
   } = useFileSystem();
 
   const { setIsLoading, isLoading } = useContext(ReaderContext);
-  const { injectBookVariables } = useInjectBookVariables();
-
+  const { injectWebviewVariables } = useInjectinjectWebviewVariables();
   const [template, setTemplate] = useState<string | null>(null);
+  const [templateUrl, setTemplateUrl] = useState<string | null>(null);
+  const [allowedUris, setAllowedUris] = useState<string | null>(null);
 
   useEffect(() => {
     (async () => {
       setIsLoading(true);
 
+      let jzipJsFileUri;
+      let epubJsFileUri;
+
+      const jzipDownloadResumable = FileSystem.createDownloadResumable(
+        'https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.5/jszip.min.js',
+        `${FileSystem.documentDirectory}jszip.min.js`,
+        {},
+        () => {}
+      );
+
+      try {
+        jzipJsFileUri = ((await jzipDownloadResumable.downloadAsync()) as FileSystemDownloadResult)
+          .uri;
+      } catch (e) {
+        throw new Error('failed to download jzip js file');
+      }
+
+      const epubjsDownloadResumable = FileSystem.createDownloadResumable(
+        'https://cdn.jsdelivr.net/npm/epubjs/dist/epub.min.js',
+        `${FileSystem.documentDirectory}epub.min.js`,
+        {},
+        () => {}
+      );
```

Comment:
Maybe we can have those files locally, next to src/template.ts? And use FileSystem to load them?